crates/bevy_window/src/window.rs

```diff
@@ -758,7 +782,7 @@ pub enum PresentMode {
     Fifo = 4, // NOTE: The explicit ordinal values mirror wgpu.
 }
 
-/// Specifies how the alpha channel of the textures should be handled during compositing.
+/// Specifies how the alpha channel of the textures should be handled during compositing in, for a [`Window`].
```

Comment:
This is very unclear to me now. I can't figure out what the "in" is supposed to mean.

Reply:
It's a typo, oops

Not sure how to word this, just remove the "in"?

Reply:
solved by https://github.com/bevyengine/bevy/pull/8858/commits/4adea2a97fd75e013fce9cdf118b3aa6edc8e8d3